### PR TITLE
Add more printer setup parameter to Parameter Settings Menu, add feedback sounds, add error display during icon update & other bug fixes

### LINKED ISSUE #432: Additional settings in parameters menu?
Would it be possible to add additional items to the parameters menu? Specifically Jerk and Acceleration. I tried to do this on my own and got everything added but my values  all keep showing 0. I program industrial equipment so I have little to no knowledge of C language and am at a dead end. I think this could be a useful addition for a number of users.

### LINKED ISSUE #434: TFT35 V3.0: counter values shown instead of position values after home
The main page shows counter values instead of position values after home command.
Steps to reproduce the issue:
* build BIGTREE_TFT35_V3.0 using VSCode and PlatformIO
* copy .pio/build/BIGTREE_TFT35_V3.0/BIGTREE_TFT35_V3.0.25.2.bin to SDCard
* copy ./Copy to SD Card root directory to update -  Unified Menu Material theme/TFT35 folder to SDCard
* insert the SDCard into TFT35 and flash the new version
* go to Menu->Movement->Home->Home
* wait the homing
* got to Menu->Movement->Move and perform any movement

Expected behaviour:
PositionOK

Actual behaviour:
PositionActual

The first occurence of this issue appears at commit a0e7296196bfdaf7530edcae01e7008fbdccb836

### LINKED ISSUE #466: [BUG] BaudRate.bmp icon does not update SW V25.3
### Description

BaudRate.bmp icon does not update SW V25.3

BaudRate

### Steps to reproduce
Updated as usual

**Expected behavior**

The icon does not update, it remains from the previous firmware.

### Additional Information

Who can check how the icons are updated in the new version?
[TFT24 Iconpack 'The Round Miracle' by Acenotass for v25.3.zip](https://github.com/bigtreetech/BIGTREETECH-TouchScreenFirmware/files/4389247/TFT24.Iconpack.The.Round.Miracle.by.Acenotass.for.v25.3.zip)
[TFT28 Iconpack 'The Round Miracle' by Acenotass for v25.3.zip](https://github.com/bigtreetech/BIGTREETECH-TouchScreenFirmware/files/4389251/TFT28.Iconpack.The.Round.Miracle.by.Acenotass.for.v25.3.zip)
[TFT35 Iconpack 'The Round Miracle' by Acenotass for v25.3.zip](https://github.com/bigtreetech/BIGTREETECH-TouchScreenFirmware/files/4389252/TFT35.Iconpack.The.Round.Miracle.by.Acenotass.for.v25.3.zip)

### PR DESCRIPTION
**Pull request by guruathwal**
- Add Max Feed rate, Max Acceleration, acceleration, Probe offset, TMC bump sensitivity to Parameter Settings.
- Add Buzzer feedback sounds for Error, Echo, Success. etc.
- Increase ACK buffer size to 2048 to allow parsing of M503 reply.
- BugFix Baudrate.bmp not updating.
- Add displaying of BMP icon update success count and fail count.
- Add display of update error (BMP icon name & cause of the error).
- translate some comments into English.
- BugFix displaying stepper steps count instead of axis position.

Resolves #432, resolves #466, resolves #434

### CODE REVIEW

File: TFT/src/User/API/UI/touch_process.h [hunk 1]
```diff
@@ -13,12 +13,12 @@ enum
...
 
-#define KEY_CLICK          0x0000  // key����Ϊ u16, 16(u16 16bit) - 3(3 bits flag ) = 13 bit ���� u16 ���֧�� 2^13 = 8192 ���ֵ
-#define KEY_DOUBLE_CLICK   0x2000  //����λ�����ʶ˫����
-#define KEY_LONG_RELEASE   0x4000  //�ڶ�λ�����ʶ�������ͷŶ���
-#define KEY_LONG_CLICK     0x8000  //��һλ�����ʶ��������
-
+#define KEY_CLICK          0x0000  // key����Ϊ u16, 16(u16 16bit) - 3(3 bits flag ) = 13 bit ���� u16 ���֧�� 2^13 = 8192 ����ֵ
+#define KEY_DOUBLE_CLICK   0x2000  //����λ������ʶ˫������
+#define KEY_LONG_RELEASE   0x4000  //�ڶ�λ������ʶ�������ͷŶ���
+#define KEY_LONG_CLICK     0x8000  //��һλ������ʶ��������
```
Comment (thisiskeithb) [inline TFT/src/User/API/UI/touch_process.h]: Encoding is broken here.

Comment (guruathwal) [reply TFT/src/User/API/UI/touch_process.h]: translated the comments to English

File: TFT/src/User/API/UI/touch_process.c [hunk 2]
```diff
@@ -218,10 +219,10 @@ typedef enum
...
 }KEY_STATUS;
 
-#define KEY_DOUOBLE_SPACE        15     //�೤ʱ���ڵ�������ж��?˫��
-#define KEY_LONG_PRESS_START     200     //��������ÿ�ʼ�ж��? ���� ��ֵ
+#define KEY_DOUOBLE_SPACE        15     //锟洁长时锟斤拷锟节碉拷锟斤拷锟斤拷锟斤拷卸锟轿?双锟斤拷
+#define KEY_LONG_PRESS_START     200     //锟斤拷锟斤拷锟斤拷锟斤拷每锟绞硷拷卸锟轿? 锟斤拷锟斤拷 锟斤拷值
 
-#define KEY_LONG_PRESS_SPACE_MAX 10     //����ʱ ���÷���һ�μ��?
+#define KEY_LONG_PRESS_SPACE_MAX 10     //锟斤拷锟斤拷时 锟筋长锟斤拷梅锟斤拷锟揭伙拷渭锟街?
 #define KEY_LONG_PRESS_SPACE_MIN 2      //����ʱ ��̶�÷���һ�μ�ֵ
```
Comment (guruathwal) [inline TFT/src/User/API/UI/touch_process.c]: @thisiskeithb some of the encodings was broken already in the main repo. but try translating these lines... 😂🤣😂

Comment (thisiskeithb) [reply TFT/src/User/API/UI/touch_process.c]: I fixed encoding once in https://github.com/bigtreetech/BIGTREETECH-TouchScreenFirmware/pull/394, but there were some files uploaded from the start with bad encoding so they couldn’t be fixed/translated.

Very odd comments indeed.

**Comment by GregSKR** [comment]
Nothing important, but a bad copy/paste in boot.c line 130:
`GUI_DispString(5, PADDING, (u8 *)"Updating Logo");`
"Updating Logo" should be "Updating Icons" for the icons part...

**Comment by thisiskeithb** [comment]
> Nothing important, but a bad copy/paste in boot.c line 130:
> `GUI_DispString(5, PADDING, (u8 *)"Updating Logo");`
> "Updating Logo" should be "Updating Icons" for the icons part...

Thanks for the heads up! I've created PR #622 to address this typo.

**Pull request merged**